File: problemas/1000/1071.cs
using System;

namespace uri1071
{
    internal static class Program
    {
        private static void Main()
        {
            int x = int.Parse(Console.ReadLine());
            int y = int.Parse(Console.ReadLine());
            int menor = Math.Min(x, y) + 1;
            int maior = Math.Max(x, y) - 1;
            int soma = 0;

            for (int i = menor; i <= maior; i++)
            {
                if (i % 2 != 0)
                {
                    soma += i;
                }
            }

            Console.WriteLine(soma);
        }
    }
}

File: problemas/1000/1071.php
<?php

$x = readline();
$y = readline();
$menor = min($x, $y) + 1;
$maior = max($x, $y) - 1;
$soma = 0;

for ($i = $menor; $i <= $maior; $i++) {
    if ($i % 2 != 0) {
        $soma += $i;
    }
}

echo $soma . "\n";